pimd/pim_mroute.c

```diff
@@ -782,27 +777,6 @@ int pim_mroute_socket_enable(struct pim_instance *pim)
 	return 0;
 }
 
-int pim_mroute_socket_disable(struct pim_instance *pim)
```

Comment:
Shouldn't this function be called somewhere to close the socket and stop the thread?
It looks like the thread is never stopped even if the pim instance is deleted which may lead to a crash.
I do not expect here that you fix this, but at least, please, don't delete this function.